airbyte-ci/connectors/metadata_service/orchestrator/pyproject.toml

```diff
@@ -15,7 +15,7 @@ dagster-gcp = "^0.21.14"
 google = "^3.0.0"
 jinja2 = "^3.1.2"
 pygithub = "^1.58.0"
-metadata-service = {path = "../lib", develop = true}
+metadata-service = {git = "https://github.com/airbytehq/airbyte", subdirectory = "airbyte-ci/connectors/metadata_service/lib"}
```

Comment:
@bnchrch this is a workaround to avoid relative path errors on install of the local package. The drawback is that we're not installing metadata service from the current branch, which is not great when locally iterating on both the orchestrator and metadata service.
Let's find a different approach later if this PR fixes our current OC issue.

Reply:
Travelling right now. So my replies will be sparse. But this was why we did setup.py instead of requirements.txt 😢 

Alternatively you could hard pin numpy?